src/libmeasurement_kit/ooni/templates.cpp

```diff
@@ -93,38 +93,51 @@ void http_request(Var<Entry> entry, Settings settings, http::Headers headers,
     http::request(
         settings, headers, body,
         [=](Error error, Var<http::Response> response) {
-            Entry rr;
-            /*
-             * Note: `probe_ip` comes from an external service, hence
-             * we MUST call `represent_string` _after_ `redact()`.
-             */
-            for (auto pair: headers) {
-                rr["request"]["headers"][pair.first] =
-                    represent_string(redact(pair.second));
-            }
-            rr["request"]["body"] = represent_string(redact(body));
-            rr["request"]["url"] = settings.at("http/url").c_str();
-            rr["request"]["method"] = settings.at("http/method").c_str();
 
-            // XXX we should probably update OONI data format to remove this.
-            rr["method"] = settings.at("http/method").c_str();
```

Comment:
I don't remember history either. Anyway, I checked what ooni-probe produces, noticed it does not include that line, and then decided it was safe to ditch it and the related comment.